src/models/CompanyAuthConfig.ts

```diff
@@ -0,0 +1,7 @@
+export default interface CompanyAuthConfig {
+    oath_scope_prefix: string;
+    accountUrl: string;
```

Comment:
Do we need separate ACCOUNT URL and CHS URL? What isCHS URL needed for?

Reply:
In the enviroment variables there are these two:
```
ACCOUNT_URL=http://account.chs.local
ACCOUNT_WEB_URL=http://chs.local
```
ACCOUNT_WEB_URL was already present and is the CHS_URL in my code. I have tried using ACCOUNT_WEB for both of those variables, but it broke the authorisation journey.

Reply:
I need to understand more what CHS_URL is needed for.

Reply:
this is where they are both used:

```
return `${this.authConfig.accountUrl}/oauth2/authorise`.concat(
            '?',
            `client_id=${this.authConfig.accountClientId}`,
            `&redirect_uri=${this.authConfig.chsUrl}/oauth2/user/callback`,
            `&response_type=code`,
            `&scope=${scope}`,
            `&state=${encodedNonce}`);
```
accountURL serves as the domain of the URI we send the user to for the authorisation journey, and chsUrl is used for the callback parameter